ODLA/include/ODLA/odla_compute.h

```diff
@@ -109,23 +128,23 @@ extern ODLA_API_EXPORT odla_status ODLA_API_CALL odla_CompileComputation(
 
 //! \brief Load a computation from the file system
 /*!
-  \param file_name the file name
+  \param location the location of the computation file
   \param computation the pointer to the loaded computation object
 
   \return odla_status
 */
-extern ODLA_API_EXPORT odla_status ODLA_API_CALL
-odla_LoadComputation(const odla_char* file_name, odla_computation* computation);
+extern ODLA_API_EXPORT odla_status ODLA_API_CALL odla_LoadComputation(
+    odla_resource_location location, odla_computation* computation);
```

Comment:
The resource type is  now embedded in odla_resource_location. Is it up to API implementation to decide how many and what types of resources can be loaded?

Reply:
And for types not supported, implementation should return odla_Failure.

Reply:
yes. if the impl. doesn't support some odla_resource_location_type, it should return failure.